SimPEG/electromagnetics/stretched_exponential/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
```

Comment:
@JKutt: did you want to include the stretched exponential work in this pr? We could work through it, but it does expand the scope quite a bit

Reply:
@lheagy I entirely forgot I brought this into the branch. I would say it is out of scope. Let me remove it for now unless others are interested?

Reply:
:+1: If you are game to move it elsewhere, I think that will help keep this one scoped more tightly. Thanks @JKutt!